build-tools/packages/build-cli/package.json

```diff
@@ -38,7 +38,8 @@
     "lint:fix": "npm run prettier:fix && npm run eslint:fix",
     "prettier": "prettier --check .",
     "prettier:fix": "prettier --write .",
-    "test": "mocha --forbid-only \"test/**/*.test.ts\" --unhandled-rejections=strict",
+    "test": "npm run test:mocha",
+    "test:mocha": "mocha --forbid-only \"test/**/*.test.ts\" --unhandled-rejections=strict",
```

Comment:
Should we move the declared params here into a mocha config file instead?

Reply:
I agree, but this seems to be a common pattern in the repo, so I'll leave it as-is for consistency. Here's an example from the map package:

https://github.com/microsoft/FluidFramework/blob/89943fa76f345ccbec39b216cbf4d3fb1789a3a5/packages/dds/map/package.json#L36